src/Tests/InstrEngineTests/NaglerInstrumentationMethod/NaglerInstrumentationMethod.cpp

```diff
@@ -14,6 +14,15 @@ const WCHAR CInstrumentationMethod::TestScriptFileEnvName[] = L"Nagler_TestScrip
 const WCHAR CInstrumentationMethod::TestScriptFolder[] = L"TestScripts";
 const WCHAR CInstrumentationMethod::IsRejitEnvName[] = L"Nagler_IsRejit";
 
+void AssertLogFailure(_In_ const WCHAR* wszError, ...)
+{
+    // Since this is a test libary, we will print to standard error and fail
+    va_list argptr;
+    va_start(argptr, wszError);
+    vfwprintf(stderr, wszError, argptr);
+    throw "Assert failed";
```

Comment:
nit: did we want to throw the error rather than a generic message?

Reply:
I have another PR that is built off of this one. I can try to fix this in that one.